backend/spotihue/hue.py

```diff
@@ -9,103 +9,132 @@
 
 class HueLight(phue.Light):
     """
-    A hue light that knows whether it is a white light.
+    A class that extends phue.Light. It represents a Hue light that
+    provides information about whether it is a white (non-color) light
+    and offers access to its corresponding RGB color values.
     """
 
     def __init__(self, bridge, light_id):
         super().__init__(bridge, light_id)
         self._white_light = None
+        self._rgb = None
 
     @property
     def white_light(self) -> bool:
-        """
+        """Property method to determine whether the Hue light is a white (non-color) light.
+
         Returns:
-            bool: Whether hue light is a white (non-color) light.
+            bool: True if the light is a white light, False if it's a color light.
         """
         if self._white_light is None:
             try:
-                _ = self._get('colormode')
+                _ = self._get("colormode")
                 self._white_light = False
             except KeyError:
                 self._white_light = True
 
         return self._white_light
 
+    @property
+    def rgb(self) -> Tuple[int, int, int]:
+        """Property method to return the Hue light's RGB values.
+        Converts normalized xy values to RGB color values. The x and y
+        values are assumed to be within the range [0, 1], where x + y <= 1.
+
+        Returns:
+            Tuple[int, int, int]: The RGB value as a tuple of integers (R, G, B).
+        """
+        if self._rgb is None:
+            try:
+                x, y = self.xy
+                r = int(x * 255)
+                g = int(y * 255)
+                b = int((1 - x - y) * 255)
+                self._rgb = (r, g, b)
+            except (AttributeError, ValueError):
+                self._rgb = None
+
+        return self._rgb
+
 
 class HueBridge(phue.Bridge):
     """
-    A more modular phue.Bridge. Accounts for unreachable lights (ex. a light in the Hue app which does not
-    correspond to an actual Hue light bulb), as well as non-color lights.
+    A class that extends phue.Bridge. It represents a Hue bridge that
+    accounts for unreachable and white (non-color) lights.
     """
+
     light_class = HueLight
 
     @property
     def reachable_lights(self) -> List[HueLight]:
-        """ Retrieves bridge's reachable lights.
+        """Retrieves the Hue bridge's reachable lights.
 
         Returns:
-            List[HueLight]: list of HueLight objects.
+            List[HueLight]: List of HueLight objects.
         """
         return [light for light in self.lights if light.reachable is True]
 
-    def get_light_objects(self, mode='list'):
+    def get_light_objects(self, mode="list"):
```

Comment:
@BeeStag Should the reachable property play into which light objects are retrieved?